qgreenland/config/cfg-lock.json

```diff
@@ -1766,6 +1773,13 @@
             "ftp://sidads.colorado.edu/DATASETS/NOAA/G02135/north/monthly/geotiff/09_Sep/N_202109_concentration_v3.0.tif"
           ],
           "verify_tls": true
+        },
+        "minimum_concentration_2022": {
+          "id": "minimum_concentration_2022",
+          "urls": [
+            "ftp://sidads.colorado.edu/DATASETS/NOAA/G02135/north/monthly/geotiff/09_Sep/N_202209_concentration_v3.0.tif"
```

Comment:
We can add the 'max' concentration for 2022 (March) but this layer can't be added yet. September is still a little ways off :smile: .